index.js

```diff
@@ -1,44 +1,58 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+var counter = 1;
+var counter = 2;
+console.log(counter) 
+ // => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red") 
+const name = "Raffy"
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
+var phrase = (`Man I sure am thirsty`)
+var secondPhrase = ('Ahh that hits the spot')
+
+function drinkWater(currentThirstLevel) {
```

Comment:
This approach would work if our `currentThirsLevel` was an array that we wanted to iterate over and return something for each element of the array, but that's not quite what we're going for here:

```javascript
function drinkWater(currentThirstLevel) {
  console.log("Man, I sure am thirsty")
  currentThirstLevel -= 1
  return currentThirstLevel
}
```